.gitignore

```diff
@@ -36,3 +36,6 @@ build/
 
 # VSCode
 .vscode/
+
+#log
+.log
```

Comment:
Add 1 empty line before the file ends. In Unix world, a source code file should end with empty an empty line:
https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

Any file that you see did not end with newline, you should add newline.